packages.yml

```
@@ -1,4 +1,6 @@
 packages:
-- package: fivetran/intercom_source
-  version: [">=0.6.0", "<0.7.0"]
-
+# - package: fivetran/intercom_source
+#   version: [">=0.6.0", "<0.7.0"] 
+  - git: https://github.com/fivetran/dbt_intercom_source.git
+    revision: feature/add-conversation-passthroughs
+    warn-unpinned: false
```

To update before merging
```suggestion
 - package: fivetran/intercom_source
   version: [">=0.6.0", "<0.7.0"] 
```